python/lsst/obs/base/cameraMapper.py

```diff
@@ -1106,6 +1106,14 @@ def _getBestFilter(self, storedLabel, idFilter):
                 newLabel = list(definitions)[0].makeFilterLabel()
                 return newLabel
             elif definitions:
+                # Could this be a band name that only ever identifies one of a
+                # series of filters?
```

Comment:
I've read this comment a few times now, and it still trips me up. I'm not sure how to reword it because I still don't quite understand it.

[time passes...]

Thinking about the code more and looking at the cfht filter definitions, I think I get it: this is in reference to the variations on `ugriz` in CFHT, where multiple things have `band` but only one doesn't have `afw_name`, and this block depends on the fact that `definitions` here has already been winnowed down to only the likely matches. I'm still not sure how to phrase that, or whether it's even worth thinking about (given that this is gen2).

Reply:
Yes, and to the analogous problem with "r" and "i" in HSC. Any better wording would be appreciated.

Reply:
"Could this band name be associated with one of a series of filters that are commonly referred to by different `afw_names`? For example `band='i'` and `band='i', afw_name='i2' for two different physical filters."

Maybe?

Reply:
Too verbose. Maybe:
> Some instruments have many filters for the same band, of which one is known by band name and the others only by afw name (e.g., i, i2).

Reply:
Eh, given the indent level and the "comments are 80chars" thing, I don't see any problem with multi-line comments. Your version works, though.